config/config.template.js

```diff
@@ -1,12 +1,23 @@
 module.exports = {
-    port: 8000,
-    output: 'data',
-    githubToken: '',
-    tasks: {
-        bootstrap: {minute: 0},
-        cdnjs: {minute: 2},
-        google: {minute: 4},
-        jsdelivr: {minute: 7}
-        //jquery: {minute: 5}
+  port: 8000,
+  output: 'data',
+  githubToken: '',
+  logentriesToken: '',
+  mailFrom: {
+    service: 'Gmail',
+    auth: {
+      user: '',
+      pass: ''
```

Comment:
No way around the logentries token but for email lets use https://mandrill.com/ https://github.com/andris9/nodemailer-smtp-transport